### ISSUE TITLE
[ENHANCEMENT] `chia farm summary` should show remote harvesters

### ISSUE THREAD

**Issue by geoffp** [Chia-Network/chia-blockchain#1461]
Currently we have:
```
Farming status: Farming                                                         
Total chia farmed: 0.0                                                          
User transaction fees: 0.0                                                      
Block rewards: 0.0                                                              
Last height farmed: 0                                                           
Plot count: 80                                                                  
Total size of plots: 7.919 TiB                                                  
Estimated network space: 137.114 PiB                                            
Expected time to win: 3 days                                                    
Note: log into your key using 'chia wallet show' to see rewards for each key
```
But this is disconcerting, because it doesn’t allow you to verify that your remote harvesters are operating correctly. It could be something like this when more than one harvester node is active:
```
Farming status: Farming                                                         
Total chia farmed: 0.0                                                          
User transaction fees: 0.0                                                      
Block rewards: 0.0                                                              
Last height farmed: 0                                                           
Plot count: 80
  ├ localhost: 78
  └ 10.0.0.123: 2                                                     
Total size of plots: 7.919 TiB                                                  
Estimated network space: 137.114 PiB                                            
Expected time to win: 3 days                                                    
Note: log into your key using 'chia wallet show' to see rewards for each key
```

**Comment by dorkmo** [comment]
ive found the best thing to look at in the gui for assurance is the "last attempted proof" time stamp, and is it within a minute of current time.

**Comment by geoffp** [comment]
@dorkmo, totally. And you can inspect the logs on the farmer if you have
```yaml
log_level: INFO
```
set. But this would be a much better CLI UX.

**Comment by mariano54** [comment]
Yeah this would be a nice feature to have. Farmer would need to store the farming info and provide this in the RPC. not super complicated feature. If anyone wants to code this up, let me know and I can provide guidance.

**Comment by Dravenex** [comment]
This feature would be awesome, because right now I noticed that farming is pretty unstable. It happened to me that the GUI showed "farming" but it wasn't, as I mentioned in another issue thread. I would also like a more user friendly guide for using the CLI.

**Comment by jespino** [comment]
I can give it a try.

**Comment by hedandan1989** [comment]
great idea!

**Comment by nicezic0** [comment]
I want to know the progress of this function.

**Comment by jespino** [comment]
@nicezic0 I still working on it, but I'm struggling to have the time to do it properly, I have around 75% of the work done.

**Comment by thauburger** [comment]
@jespino thanks for the effort here! This will be super helpful.

**Comment by sargonas** [comment]
@jespino anything I can do to help? This has been annoying me all weekend since I moved to a split-system setup, and so my OCD is quite motivated to help get it resolved LOL

**Comment by eric-vader** [comment]
@jespino Thanks for the work! upvote this feature!

**Comment by L3Nerd** [comment]
@jespino May I help you?

**Comment by zhubiaogg** [comment]
I want to know the progress of this function.

**Comment by hzceo** [comment]
How is going now.So important feature.

**Comment by lvpcguru** [comment]
Looks like its in the works in this PR and another [https://github.com/Chia-Network/chia-blockchain/pull/3600](https://github.com/Chia-Network/chia-blockchain/pull/3600)

**Comment by mariano54** [comment]
Fixed by 1.2.0
